### ISSUE TITLE
Integrate font with ontology

### ISSUE THREAD

**Issue by jakebeal** [SynBioDex/SBOL-visual#98]
The [SBOL Visual font](https://github.com/SynBioDex/SBOL-visual/tree/master/font) is currently readily usable only by humans and not machines, because there is not an available mapping from glyphs to font character IDs. 

This should be readily addressable via the ontology, but somebody needs to actually add that mapping.

**Comment by goksel** [comment]
@jakebeal I would need to understand more about this issue. Can the font information be incorporated into README files or is there any description of this mapping somewhere?

**Comment by jakebeal** [comment]
The information that is needed is in the font itself. Resolving this, however, can probably be deferred until after #115, which might render it moot.  That's why this one doesn't have anybody assigned to it yet.
